pkg/jsonhttp/jsonhttp_test.go

```diff
@@ -125,6 +129,8 @@ func TestRespond_statusResponse(t *testing.T) {
 }
```

Comment:
I think the whole body within the loop can be inside a parallel `t.Run`, no?

Reply:
At this point I am trying to just add `t.Parallel()` to functions, without further optimising tests. Because that itself is different rabbit hole.